### DISCUSSION TITLE
Cloudflare Pages - fs.readFile is not a function

### DISCUSSION THREAD

**Discussion by uxbert-fe** [sergiodxa/remix-i18next#113] [Q&A]
Using the `cloudflare pages` setup, produce the following error: 

```
i18next::backendConnector: loading namespace common for language en failed TypeError: fs.readFile is not a function
```

Issue #36 kinda gives a hint to import from the build files directly, but not more info on what actually that means and/or code example.

This is a fresh setup repo that shows the issue: https://github.com/uxbert-fe/remix-18next-cf-pages

Thanks in advance

**Comment by sergiodxa** [comment]
You can't use the file system in Cloudflare so you need to use another backend.

**Comment by uxbert-fe** [comment]
Thank you.

I ended up with a similar setup to your repo: https://github.com/sergiodxa/sergiodxa.com
